### ISSUE TITLE
logcli spits out multiple 'proto: duplicate proto type registered:' messages

### ISSUE THREAD

**Issue by yakob-aleksandrovich** [grafana/loki#4910]
**Describe the bug**
when running the logcli binary (downloaded from https://github.com/grafana/loki/releases/download/v2.4.1/logcli-linux-amd64.zip), it spits out a flurry of messages, all starting with '<timestamp>  I | proto: duplicate proto type registered:', finishing with the actual URL that it used to query Loki.

**To Reproduce**
1. Follow manual docker procedure outlined [here](https://grafana.com/docs/loki/latest/installation/docker/)
2. Run basic command from logcli: `./logcli labels`
Output:
```
2021-12-10 16:18:04.007693 I | proto: duplicate proto type registered: purgeplan.DeletePlan
2021-12-10 16:18:04.007741 I | proto: duplicate proto type registered: purgeplan.ChunksGroup
2021-12-10 16:18:04.007746 I | proto: duplicate proto type registered: purgeplan.ChunkDetails
2021-12-10 16:18:04.007750 I | proto: duplicate proto type registered: purgeplan.Interval
2021-12-10 16:18:04.018092 I | proto: duplicate proto type registered: grpc.PutChunksRequest
2021-12-10 16:18:04.018108 I | proto: duplicate proto type registered: grpc.GetChunksRequest
...
2021-12-10 16:40:50.841507 I | proto: duplicate proto type registered: storage.Entry
2021-12-10 16:40:50.841515 I | proto: duplicate proto type registered: storage.ReadBatch
http://localhost:3100/loki/api/v1/labels?end=1639150850842286965&start=1639147250842286965
```

**Expected behavior**
I expected to just get the labels that exist in the sytem:
```
__name__
application
job
...
```

**Environment:**
 - Infrastructure: Laptop, Linux Mint 20.2
 - Deployment tool: Manual

**Screenshots, Promtail config, or terminal output**
Configuration is completely bog standard, straight from github
```
wget https://raw.githubusercontent.com/grafana/loki/v2.4.1/cmd/loki/loki-local-config.yaml -O loki-config.yaml
docker run --name loki -v $(pwd):/mnt/config -p 3100:3100 grafana/loki:2.4.1 -config.file=/mnt/config/loki-config.yaml
wget https://raw.githubusercontent.com/grafana/loki/v2.4.1/clients/cmd/promtail/promtail-docker-config.yaml -O promtail-config.yaml
docker run -v $(pwd):/mnt/config -v /var/log:/var/log --link loki grafana/promtail:2.4.1 -config.file=/mnt/config/promtail-config.yaml
```

**Comment by jeschkies** [comment]
Could this be a duplicate of #3104?

**Comment by yakob-aleksandrovich** [comment]
#3104 only mentions ingester.Series, which is fixed. The problem that I am describing is similar, but not the same: I get quite a bit more duplicates, for different libraries/functions(?). So, it's similar, but **not** the same, and the solution will have to be executed along the same lines, I assume.

**Comment by chaudum** [comment]
The problem is already resolved on `main` and will be released with the next minor (2.5) release.

If you want, you can build `logcli` from `main` by invoking `make logcli`.
